kafka/zoo_sample.cfg

```diff
@@ -0,0 +1,28 @@
+# The number of milliseconds of each tick
```

Comment:
is this an accident?

Reply:
Since I stopped extracting zookeeper into the image as it's a part of Kafka itself, I copied this file to only add it. But realized even if I don't copy zookeeper, I could still curl it like before just to get the config file without including it in the repo.

Reply:
ok